Update readmes to match common design (Map category) 2/4 (#466)
Esri/arcgis-runtime-samples-java: map/generate-offline-map-with-local-basemap/README.md

# Generate offline map with local basemap

> Use the `OfflineMapTask` to take a web map offline, but instead of downloading an online basemap, use one which is already on the device.

Reviewer: We should avoid using class names in this section, would suggest something like the following:
```suggestion
Take a web map offline using a local basemap on the device.
```

## How to use the sample

1. Click on the "Take Map Offline" button.

> 2. You will be prompted to choose whether you wish to download the online basemap or use the "naperville_imagery.tpk" basemap (which is already saved in the samples-data directory).

Reviewer 2: Looks like the old one was an exception, but we usually don't have a numbered list in this section, just paragraphs.

Reviewer: yeh we should turn this into a paragraph rather than numbered points. must have also been overlooked in the common sample revamp.
